app/deliver_grant_funding/routes.py

```diff
@@ -180,6 +182,30 @@ def list_grants() -> Response | str:
     return render_template("deliver_grant_funding/grant_list.html", grants=grants)
 
 
+@deliver_grant_funding_blueprint.route("/users/<uuid:grant_id>", methods=["GET", "POST"])
```

Comment:
There's no post here?

```suggestion
@deliver_grant_funding_blueprint.route("/users/<uuid:grant_id>", methods=["GET"])
```